install_config/install/advanced_install.adoc

```diff
@@ -1745,6 +1745,22 @@ endif::[]
 These settings create a persistent volume that is attached to the OAB's etcd
 instance during cluster installation.
 
+[[configuring-oab-local-apb-devel]]
+==== Configuring the OpenShift Ansible Broker for Local APB Development
+
+In order to do xref:../../apb_devel/index.adoc#apb-devel-intro[APB development]
+with the OpenShift Container Registry in conjunction with the OAB, a whitelist
+must be defined. By default, the whitelist is empty so that a user cannot add
+APB images to the broker without a cluster administrator configuring the broker.
+
```

Comment:
Is this task required to use OAB? If not, should we mention this? 
_You can use OAB with any registry, as configured in...._

Reply:
Yes a whitelist must be defined or the Broker will completely ignore APBs. End-users will simply not see any APBs and it is not apparent what is going wrong.